lib/name_drop/configuration.rb

```diff
@@ -15,18 +15,4 @@ def initialize
       @access_token = 'NotARealAccessToken'
     end
   end
-
-  class << self
```

Comment:
I'm on board with that. Only thing I'd request is that we at least have this have in the main `name_drop.rb`:
```ruby
module NameDrop
end
```